sharding/src/main/scala/com/lightbend/akka/bench/sharding/scalability/ActorCountingBenchmarkMaster.scala

```diff
@@ -0,0 +1,62 @@
+package com.lightbend.akka.bench.sharding.scalability
+
+import java.util
+import java.util.concurrent.TimeUnit
+
+import scala.concurrent.duration._
+import akka.actor.{ Actor, ActorLogging, ActorRef }
+import akka.cluster.Cluster
+
+class ActorCountingBenchmarkMaster extends Actor with ActorLogging {
+  val config = context.system.settings.config
+  val addActorsBatch = config.getInt("shard-bench.add-actors-batch")
+  val addActorsInterval = config.getDuration("shard-bench.add-actors-interval", TimeUnit.MILLISECONDS).millis
+  
+  val cluster = Cluster(context.system)
+  
+  // number of actors we sent a message to (so they should start in sharding)
+  var totalStartingActorsInSharding = 0
+  // number of actors who have not yet replied back that they've started
+  var pendingAliveConfirmation = 0
+  var totalAliveConfirmedActors = 0
+  val timesWhenWeStartedBatch = new util.ArrayDeque[Long](10) 
+  
+  override def preStart(): Unit = {
+    context.system.scheduler.schedule(addActorsInterval, addActorsInterval, self, AddMoreActors)
+  }
+  
+  val sharding: ActorRef = ActorCountingEntity.proxy(context.system)
+  
+  override def receive: Receive = {
+    case ActorCountingEntity.Ready(startTime) =>
+      // not really goal of this benchmark though: 
+      // if (log.isDebugEnabled)
+      //   log.debug("Actor {} took [{} ns] to initialize and reply-back", sender().path.name, startTime - System.nanoTime())
+      
+      pendingAliveConfirmation -= 1
+      totalAliveConfirmedActors += 1
+      if (totalAliveConfirmedActors % addActorsBatch == 0) {
+        val timeToStartBatch = (System.nanoTime() - timesWhenWeStartedBatch.pop()).nanos.toMillis
+        log.info("FINISHED INITIALIZING ANOTHER {} ACTORS (TOTAL: {}) IN [{} ms]!!!", addActorsBatch, totalAliveConfirmedActors, timeToStartBatch)
```

Comment:
so we know how long it took to start the 100 actors, as well as how many in total are active now